test/e2e/testdata/template-failed-metric.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: dag-task-test
+spec:
+  entrypoint: dag-task
+  templates:
+  - name: dag-task
+    metrics:
+        prometheus:
+        - counter:
+            value: "1"
+          # help: DAG Task Workflow succeeded
+          # name: dagtask_workflow_success
+          # when: '{{status}} == Succeeded'
+          help: DAG Task Workflow failed
+          name: dagtask_workflow_failure
```

```suggestion
          help: Task failed
          name: task_failure
```

use less confusing and simpler naming/help text